docs/docs/cloud/cloud-cli.mdx

```diff
@@ -48,6 +48,90 @@ meltano cloud config env delete TAP_GITHUB_AUTH_TOKEN
 
 See the [reserved variables](/cloud/platform#reserved-variables) docs for more details on variables that are reserved for use by Meltano Cloud.
 
+### `notification`
+
+The `notification` command provides a way for managing your [Meltano Cloud notifications](concepts#meltano-cloud-notifications).
+
+#### `list`
+
+List all notifications configured for your current default project or organization.
+
+```sh
+$ meltano cloud config notification list
+╭─────────────────────┬─────────┬──────────╮
+│ Recipient           │ Type    │ Status   │
+├─────────────────────┼─────────┼──────────┤
+│ https://melty       │ webhook │ active   │
+│ dragon@meltano.com  │ email   │ active   │
+╰─────────────────────┴─────────┴──────────╯
+```
+
+#### `set`
+
+Set a new notification or override an existing one.
+
+##### Case-Sensitivity in Values
+
+Take note that all the values in the commands and configuration settings are case-sensitive. This means job_created, Job_Created, and JOB_CREATED would be considered as different events. Please ensure to use the correct casing to avoid configuration issues or unexpected behaviors.
+
+
+Interactive mode
+```sh
+meltano cloud config notification set <type>
+```
+
+Non-interactive mode:
+```sh
+meltano cloud config notification set <type> --recipient <target_recipient> --filter <filter>
+````
+
+**`options`**
+
+`--recipient` (Required): The recipient to send the notification to.
+
+`--filter`: An optional filter you want to check against, you can supply multiple filters which will all be joined as an OR expression.
```

Comment:
What is the default filter? If I don't set anything what does the payload look like?

Reply:
Default filter is just an empty list meaning that there are no filters and you will always receive the notification. I believe payload just includes `filter: []`, when returned from the list, it just shows an empty list as well

Reply:
I like the idea of showing `Filter: N/A` or something like that. and then if there is a filter it's obvious what it is. 

Basically if someone has 10 notifications set up, how do they know which ones are for failed notifications or not?

Reply:
This looks good